File: Frontend/src/infrastructure/dtos/vvn/fullVVNCreateDto.ts
import type { LoadCargoManifestDto } from "../cargo-manifest/loadCargoManifestDto";
import type { UnloadCargoManifestDto } from "../cargo-manifest/unloadCargoManifestDto";
import type { VVNCreateDto } from "./vvnCreateDto";

export type FullVVNCreateDto = VVNCreateDto & {
  cargoLoadManifestDTO: LoadCargoManifestDto;
  cargoUnloadManifestDTO: UnloadCargoManifestDto;
};

File: JWP_API/JadeWesserPort/DTOs/VVNDTOs/FullVVNCreateDTO.cs
using JadeWesserPort.DTOs.CargoManifestDTOs;

namespace JadeWesserPort.DTOs.VVNDTOs;

public class FullVVNCreateDTO : VVNCreateDTO
{
    public LoadCargoManifestDTO CargoLoadManifestDTO { get; set; } = null!;
    public UnloadCargoManifestDTO CargoUnloadManifestDTO { get; set; } = null!;
}
